packages/react/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@segment/analytics-react",
+  "version": "0.0.0",
```

Comment:
I think this should match what's in npm, so `1.0.0-rc`.

Reply:
I might hold off on that.  When this gets published (1.0) as part of changeset, it will auto bump to 1.0 -- I don't 100% know what will happen if we add "rc" =).  I haven't looked into the prerelease workflow. https://github.com/changesets/changesets/blob/main/docs/prereleases.md

Reply:
@silesky did some digging into how changeset will version prereleases - I think for now we should mark this as `"private": true` to prevent changeset from accidentally publishing it, and then we can manually release it as we need until we mark it GA.

Reply:
Do we need to wait for something else before this goes into GA?